src/components/task-card/components/header/styles.module.css

```diff
@@ -0,0 +1,22 @@
+.taskCardHeader {
+  display: grid;
+  grid-template-columns: 3fr 5fr 2fr 2fr;
+  gap: 16px;
+  padding: 8px var(--Spacing-24, 24px);
+  align-items: flex-start;
+
+  color: var(--text-secondary);
+  font-feature-settings:
+    'clig' off,
+    'liga' off;
+  /* caption_medium */
+  font-family: 'Instrument Sans';
+  font-size: 0.688rem;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 1rem; /* 145.455% */
+
+  border-bottom: 1px solid var(--stroke-tabs-sections);
+  border-radius: 10px 10px 0 0;
+  background: #fbfbfe;
```

Comment:
no use of colours which are not part of the design system, if it's missing ask the design team to add it